doc/dev/developer_guide/basic-workflow.rst

```diff
@@ -154,73 +208,122 @@ Create a branch for your bugfix:
    git checkout -b fix_1
    git push -u origin fix_1
 
-This creates a local branch called ``fix_1`` in our GitHub fork. At this point,
-the ``fix_1`` branch is identical to the ``master`` branch, but not for long!
-You are now ready to modify the code.  Be careful to always run `git checkout
-master` first, otherwise you may find commits from an unrelated branch mixed
-with your new work.
+The first command (git checkout master) makes sure that the bugfix branch
```

Comment:
Aren't we renaming `master` to `main`?

Reply:
When it happens, I'll alter the documentation to reflect it.